### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by wilfrem** [TypeStrong/atom-typescript#128]
[Enter steps to reproduce below:]

1. open typescript file.
2. create new file at another tab("untitled" tab opened)
3. back typescript file and press F2 and rename identifier.

**Atom Version**: 0.182.0
**System**: Unknown Windows Version
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package, v0.69.0

### Stack Trace

Uncaught TypeError: Cannot read property 'split' of undefined

```
At C:\Users\kazuki_yasufuku\.atom\packages\atom-typescript\lib\main\tsconfig\tsconfig.js:201

TypeError: Cannot read property 'split' of undefined
  at Object.consistentPath (C:\Users\kazuki_yasufuku\.atom\packages\atom-typescript\lib\main\tsconfig\tsconfig.js:201:20)
  at addConsistentlyToMap (C:\Users\kazuki_yasufuku\.atom\packages\atom-typescript\lib\main\atom\atomUtils.js:39:22)
  at Array.forEach (native)
  at Object.getEditorsForAllPaths (C:\Users\kazuki_yasufuku\.atom\packages\atom-typescript\lib\main\atom\atomUtils.js:41:19)
  at Object.renameView.panelView.renameThis.onCommit (C:\Users\kazuki_yasufuku\.atom\packages\atom-typescript\lib\main\atom\commands.js:95:31)
  at atom-text-editor.<anonymous> (C:\Users\kazuki_yasufuku\.atom\packages\atom-typescript\lib\main\atom\views\renameView.js:29:35)
  at atom-text-editor.jQuery.event.dispatch (C:\Users\kazuki_yasufuku\AppData\Local\atom\app-0.182.0\resources\app\node_modules\jquery\dist\jquery.js:4430:9)
  at atom-text-editor.elemData.handle (C:\Users\kazuki_yasufuku\AppData\Local\atom\app-0.182.0\resources\app\node_modules\jquery\dist\jquery.js:4116:28)

```

### Commands

```
     -0:44.5 typescript:rename-variable (atom-text-editor.editor)
     -0:39.8 core:confirm (atom-text-editor.editor.mini)
     -0:12.9 typescript:rename-variable (atom-text-editor.editor)
     -0:11.1 core:move-right (atom-text-editor.editor.mini)
     -0:05.9 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-typescript, v0.69.0
autocomplete-plus, v2.2.0
linter, v0.11.1
project-manager, v1.15.5

# Dev
No dev packages
```

**Comment by basarat** [comment]
Note to self : filter out editors that don't have paths.

**Comment by basarat** [comment]
Thanks for trying the package and sorry about the inconvenience. Fixed in `v0.71.0` :heart:
